deps/rabbit/src/rabbit_disk_monitor.erl

```diff
@@ -227,11 +227,87 @@ get_disk_free(Dir) ->
 get_disk_free(Dir, {unix, Sun})
   when Sun =:= sunos; Sun =:= sunos4; Sun =:= solaris ->
     Df = os:find_executable("df"),
-    parse_free_unix(rabbit_misc:os_cmd(Df ++ " -k " ++ Dir));
+    parse_free_unix(run_cmd(Df ++ " -k " ++ Dir));
 get_disk_free(Dir, {unix, _}) ->
     Df = os:find_executable("df"),
-    parse_free_unix(rabbit_misc:os_cmd(Df ++ " -kP " ++ Dir));
+    parse_free_unix(run_cmd(Df ++ " -kP " ++ Dir));
 get_disk_free(Dir, {win32, _}) ->
+    % Dir:
+    % "c:/Users/username/AppData/Roaming/RabbitMQ/db/rabbit2@username-z01-mnesia"
+    case win32_get_drive_letter(Dir) of
+        error ->
+            rabbit_log:warning("Mnesia directory is not in the expected format (C:): '~p'", [Dir]),
```

Comment:
Should we mention that `C:` actually means a drive letter, so a specific disk is not a requirement? Or would it be obvious to a Windows user?

Reply:
It _should_ be obvious, but I can make the message clearer.